edit/lint.js

```diff
@@ -3,7 +3,7 @@
 /* global loadScript require CSSLint stylelint */
 'use strict';
 
-loadLinterAssets();
+onDOMready().then(loadLinterAssets);
```

Comment:
Why? It's already queued correctly (onDOMscripted will resolve after onDOMready) and works in the master branch.

Reply:
`onDOMscripted` does resolve after `onDOMready`, but the scripts are injected immediately when `onDOMscripted` is called:
```js
// original onDOMready()
loadScript(scripts).then(onDOMready);

// I think this is what we want
onDOMready().then(() => loadScript(scripts));
```

Reply:
Well, yeah, but previously we could omit onDOMready as it was incorporated in onDOMscripted.

Reply:
Oops, typo. It should be *original onDOMscripted*.